search/src/main/kotlin/com/alfresco/content/search/ListViewFilterChips.kt

```diff
@@ -62,7 +62,13 @@ class ListViewFilterChips @JvmOverloads constructor(
     private fun String.take20orDefault(): String {
```

Comment:
Function name should be generic

Reply:
it's default method name is take to get the string so I make for 20 character like this

Reply:
done